apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -1,20 +1,22 @@
 import { GraphPointer } from 'clownface'
-import { rdfs, rdf, hydra } from '@tpluscode/rdf-ns-builders'
+import { rdfs, rdf, hydra, dcterms } from '@tpluscode/rdf-ns-builders'
 import { cc } from '@cube-creator/core/namespace'
 import { ResourceStore } from '../../ResourceStore'
-import { cubeProject, csvMapping } from '../identifiers'
+import { cubeProject, csvMapping, creatorUser } from '../identifiers'
 
 interface CreateProjectCommand {
   resource: GraphPointer
   store: ResourceStore
+  user: string
```

Comment:
I would also make the parameter a named node so that the project creation does not create new resources but depends on a concrete value

```suggestion
  creator: NamedNode
```

Maybe trivial but this way the user id becomes an opaque input. Even if we indirect the URI creation with `identifiers`